tox.ini

```diff
@@ -90,6 +89,7 @@ allowlist_externals = touch
 deps =
     sphinx~=3.5
     sphinx-autodoc-typehints~=1.11
+    jinja2~=3.0.0
```

Comment:
What is jinja2 used for?

Reply:
There's an issue with sphinx 3.5 and newer versions of jinja2 (`>3.1`), so here we keep it at a compatible version.

Reply:
Does it break things to use the latest Sphinx, that seems to solve this issue? If yes, we should probably keep this link handy in a comment next to this line?

Reply:
I haven't tried, but I can test it

Reply:
I tested it and it looks good AFAICT. See: https://michael-the1.github.io/diepvries/

Reply:
Looks great indeed!